iptables/table.go

```diff
@@ -289,7 +341,14 @@ func (t *Table) UpdateChains(chains []*Chain) {
 
 func (t *Table) UpdateChain(chain *Chain) {
 	t.logCxt.WithField("chainName", chain.Name).Info("Queueing update of chain.")
+	oldChain := t.chainNameToChain[chain.Name]
+	oldNumRules := 0
+	if oldChain != nil {
```

Comment:
Perhaps be more idiomatic by moving the oldChain assignment into the previous line?